app/src/main/java/ch/epfl/sweng/SDP/game/RankingFragment.java

```diff
@@ -132,7 +133,7 @@ public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                 int trophiesForUser = trophies[usernames.indexOf(account.getUsername())];
 
                 Boolean won = usernames.get(0).equals(account.getUsername());
-                updateUserStats(rankForUser, trophiesForUser, won);
+                updateUserStats(Math.max(rankForUser, 0), trophiesForUser, won);
```

Comment:
Ranking cannot be negative because ranking is the number of stars you earn. But in my implementation I set negative ranking to players that are disconnected in order to treat them differently. But in the end I do not want to show a negative ranking thus the max function